src/utilities/css/ContextMenu.css

```
@@ -141,12 +141,14 @@
 }
 
 
-.ciview-menu-bar{
-  flex:0 0 auto;
-  padding:3px 20px;
-  background-image: var(--menu-gradient);
+.ciview-menu-bar {
+  flex: 0 0 auto;
+  padding: 5px 25px;
+  background-color: var(--menu_bar_color);
   color: var(--text_color);
-
+  margin-top: 5px; /* Add a top margin */
+  border: 2px solid var(--border_menu_bar_color);
+  box-shadow: 2px 4px 8px rgba(0, 0, 0, 0.2); 
```

I think this looks good on `border-bottom`, other borders seem to be what was distracting to me particularly in dark-mode (but also subtly in light mode)
```suggestion
  border-bottom: 2px solid var(--border_menu_bar_color);
```

---

not sure how that `box-shadow` removal got in there, not intended.